app/src/main/java/org/mozilla/reference/browser/CustomTabActivity.kt

```diff
@@ -4,4 +4,15 @@
 
 package org.mozilla.reference.browser
 
-class CustomTabActivity : BrowserActivity()
+import android.os.Bundle
+import org.mozilla.reference.browser.ext.components
+
+class CustomTabActivity : BrowserActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+
+        // If a Custom Tab is restored by the OS after low memory, we need to process the intent again.
+        components.utils.intentProcessor.process(intent)
```

Comment:
I fail to understand why this activity would be recreated with the custom tab intent instead of the intent **this** activity was created with 🤔